htsget_ingest.py

```diff
@@ -10,19 +10,33 @@
 from urllib.parse import urlparse
 from clinical_etl.schema import openapi_to_jsonschema
 import jsonschema
+from candigv2_logging.logging import CanDIGLogger
+
+
+logger = CanDIGLogger(__file__)
 
 
 CANDIG_URL = os.getenv("CANDIG_URL", "")
 HTSGET_URL = os.getenv("HTSGET_URL", f"{CANDIG_URL}/genomics")
 DRS_HOST_URL = "drs://" + CANDIG_URL.replace(f"{urlparse(CANDIG_URL).scheme}://","") + "/genomics"
+KATSU_URL = os.environ.get("KATSU_URL")
+IS_TESTING = os.getenv("IS_TESTING", False)
 
 
-def link_genomic_data(headers, sample):
+def link_genomic_data(sample, do_not_index=False):
     url = f"{HTSGET_URL}/ga4gh/drs/v1/objects"
     result = {
         "errors": []
     }
 
+    # Use service token to authenticate this with htsget
+    headers = {}
+    if not IS_TESTING:
+        headers = {
+            "X-Service-Token": create_service_token(),
+            "Content-Type": "application/json"
+        }
+
```

Comment:
I'm confused -- why is problematic while testing?

Reply:
In the local repo's pytest, it's not a container environment and so it doesn't have access to Vault and Opa.